extensions/interactions/FractionInput/directives/oppia-interactive-fraction-input.component.ts

```diff
@@ -188,8 +188,9 @@ export class InteractiveFractionInputComponent implements OnInit, OnDestroy {
           fraction, this.fractionInputRulesService);
       }
     } catch (parsingError) {
-      let error = parsingError as Error;
-      this.errorMessageI18nKey = error.message;
+      if (parsingError instanceof Error) {
+        this.errorMessageI18nKey = parsingError.message;
+      }
```

Comment:
I think we should rethrow the error when it is not `Error`

Reply:
if you are saying, to set `errorMessageI18nKey` something, then it is possible. And if you are saying to `throw parsingError` then it may break flow of this function and further progress, as this whole function works to check invalidity. If something is invalid we make `isvalid = false`. Which is use further in isAnswerValid() function, which is passed to parent service/component.

Reply:
@vojtechjelinek PTAL.

Reply:
I think it is fine if that breaks it. As this will most probably be caught in tests.

Reply:
Done